ckan/logic/action/create.py

```diff
@@ -296,6 +296,12 @@ def resource_create(context, data_dict):
 
     upload = uploader.get_resource_uploader(data_dict)
 
+    if not 'size' in data_dict and hasattr(upload, 'upload_file'):
```

Comment:
Yes, please. Your IDE/linter should be yelling at you to fix it :) At lot of the old CKAN code does not follow PEP8 and it's cleaned up as we go.